src/main/scala/chisel3/util/Counter.scala

```diff
@@ -113,7 +112,6 @@ object Counter {
     * @return tuple of the counter value and whether the counter will wrap (the value is at
     * maximum and the condition is true).
     */
-  @chiselName
   def apply(cond: Bool, n: Int): (UInt, Bool) = {
```

Comment:
does returning Tuples of hardware work with the plugin?

Reply:
I don't think so, but I'm not sure how to return a Tuple that extends `AffectsChiselPrefix`

Reply:
@jackkoenig Any ideas?

Reply:
@mwachs5 I created a module that calls this function, with and without `chiselName` and there does not seem to be any difference in the naming of the resulting firrtl. I don't believe that `chiselName` is doing anything here.